daemon/cmld.c

```diff
@@ -940,20 +940,34 @@ cmld_container_create_clone(container_t *container)
 }
 
 container_t *
-cmld_container_create_from_config(const char *config, size_t config_len)
+cmld_container_create_from_config(const uint8_t *config, size_t config_len)
 {
 	ASSERT(config);
 	ASSERT(config_len);
-	ASSERT(0);
-	return NULL;
+	char *path = mem_printf("%s/%s", cmld_path, CMLD_PATH_CONTAINERS_DIR);
```

Comment:
for another mem_printf in this patch we did a if null error check...

Reply:
check everywhere else in patches

Reply:
you are right if null container_new would run into an assertion. Fixed it by         IF_NULL_RETVAL(path, NULL);